tests/Fixtures/TestBundle/Document/MultiRelationsNestedPaginated.php

```diff
@@ -17,7 +17,7 @@
 use ApiPlatform\Metadata\GraphQl\QueryCollection;
 use Doctrine\ODM\MongoDB\Mapping\Annotations as ODM;
 
-#[ApiResource(graphQlOperations: [new QueryCollection(nested: true)])]
+#[ApiResource(graphQlOperations: [new QueryCollection(provider: self::class, nested: true)])]
```

Comment:
you want to force the provider to `null`?

Reply:
Exactly. I found no way how to do that. Setting the provider explicitely to `null` also resulted that the operation had the default provider. So my "trick" was to use the entity itself as a provider. I am open to other ideas or contribution.

Reply:
Like the idea, will change it soon

Reply:
Changed it and work like a charm.Thx